### DISCUSSION TITLE
Fallback for the disabled local auth

### DISCUSSION THREAD

**Discussion by fabioteixei** [goauthentik/authentik#7950] [Q&A] [answered]
Hello

I have followed this guide to disable local auth in order to automaticly redirect all authentication to Azure AD: https://blog.lrvt.de/authentik-traefik-azure-ad/#disable-local-auth

It's working great, when my users access an aplication, Authentik quickly apears and they are redirected to Azure AD auth, the usres are authenticated with Azure AD and them back to the aplication.

I just want to know if theres any way to have some kind of fall back, moslty for me as Admin, to enter Authentik Dashboard in a case when Azure AD is not avaliable for an reason or another? Create another outpost maybe, i dont know.

Thanks

**Comment by BeryJu** [comment, marked as answer]
You can either create a separate login flow that is configured differently, that can be used to login, or as a fallback you can also use https://goauthentik.io/docs/troubleshooting/login

**Reply by fabioteixei** [reply]
Thanks @BeryJu that will do the trick since i need that only for troubleshooting process. I just have only one sugestion is to improve the documentation, stating that the recovery link is single use, once is used it cannot be re-used so we need a new one for every time we need to log in.

Thanks
